k8s/demo/common/divorce/div-da.yaml

```diff
@@ -16,7 +16,7 @@ spec:
     nodejs:
       replicas: 2
       useInterpodAntiAffinity: true
-      image: hmctspublic.azurecr.io/div/da:prod-d0c9b1d-20211203120928 #{"$imagepolicy": "flux-system:div-da"}
+      image: hmctspublic.azurecr.io/div/da:pr-161-8ba99a4-20211206094017 #{"$imagepolicy": "flux-system:div-da"}
```

Comment:
I believe you want demo added to the image policy here, like 
image: hmctspublic.azurecr.io/div/da:pr-161-8ba99a4-20211206094017 #{"$imagepolicy": "flux-system:demo-div-da"}